plugins/LanguagesManager/API.php

```diff
@@ -275,6 +275,45 @@ public function setLanguageForUser($login, $languageCode)
         return true;
     }
 
+    /**
+     * Returns whether the user uses 12 hour clock
+     *
+     * @param string $login
+     * @return string
+     */
+    public function uses12HourClockForUser($login)
+    {
+        if ($login == 'anonymous') {
+            return false;
+        }
+
+        Piwik::checkUserHasSuperUserAccessOrIsTheUser($login);
+
+        $lang = $this->getModel()->uses12HourClock($login);
+
+        return $lang;
+    }
+
+    /**
+     * Returns whether the user uses 12 hour clock
+     *
+     * @param string $login
+     * @param bool $use12HourClock
+     * @return string
+     */
+    public function set12HourClockForUser($login, $use12HourClock)
+    {
+        if ($login == 'anonymous') {
```

Comment:
At some point we should create a constant for `anonymous` but it's not related to your PR :) It will be more like a reminder for myself :)